src/software/ai/hl/stp/tactic/transition_conditions.cpp

```diff
@@ -22,3 +24,40 @@ bool compareAngles(const Angle& angle1, const Angle& angle2, const Angle& ANGLE_
 {
     return angle1.minDiff(angle2) <= ANGLE_THRESHOLD;
 }
+
+bool isRobotReadyToChick(const Robot& robot, const Point& ball_position,
+                         const Angle& chick_direction, const Angle& ANGLE_THRESHOLD)
```

Comment:
why is `ANGLE_TRESHOLD` in this casing for a function param?

Reply:
I followed the general notation of this file. I think it's in caps to kind of indicate that it's kind of like a constant since it has a default value?! not fully sure.